### PR TITLE
Update freeplane to 1.7.5

### PR DESCRIPTION
**Pull request by reagle**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/freeplane.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask 'freeplane' do
-  version '1.7.3'
-  sha256 'd993a3cc472dc4d1c50715fa4d63da93b1ef54712cfd4e13fadb793d666a0374'
+  version '1.7.5'
+  sha256 '3095b390a2ffd0d8a9466d18e983f90a8e603c44839da5394ac96d99f4a91102'
 
   # downloads.sourceforge.net/freeplane was verified as official when first introduced to the cask
-  url "https://downloads.sourceforge.net/freeplane/freeplane%20stable/freeplane_app_jre-#{version}.dmg"
+  url 'https://downloads.sourceforge.net/freeplane/freeplane%20stable/freeplane_app_jre-1.7.5-hotfix1.dmg'
```
Comment (reitermarkus) [inline Casks/freeplane.rb]: The version should still be interpolated here. You can just use `1.7.5-hotfix1` as the `version`.

Comment (reagle) [reply Casks/freeplane.rb]: I was wondering about that and understand your suggestion, but this was my first time using cask-repair and am not sure how to change it now.

Comment (reitermarkus) [reply Casks/freeplane.rb]: Either close this PR and submit a new one or amend the changes to this one.

Comment (reagle) [reply Casks/freeplane.rb]: I think I did.

File: Casks/freeplane.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 cask 'freeplane' do
-  version '1.7.3'
-  sha256 'd993a3cc472dc4d1c50715fa4d63da93b1ef54712cfd4e13fadb793d666a0374'
+  version '1.7.5-hotfix1'
+  sha256 '3095b390a2ffd0d8a9466d18e983f90a8e603c44839da5394ac96d99f4a91102'
 
   # downloads.sourceforge.net/freeplane was verified as official when first introduced to the cask
-  url "https://downloads.sourceforge.net/freeplane/freeplane%20stable/freeplane_app_jre-#{version}.dmg"
+  url 'https://downloads.sourceforge.net/freeplane/freeplane%20stable/freeplane_app_jre-#{version}.dmg'
```
Comment (reitermarkus) [inline Casks/freeplane.rb]: The double quotes need to stay.

Comment (reagle) [reply Casks/freeplane.rb]: Okay.

**Comment by reitermarkus** [comment]
Thanks!

**Pull request merged**